### PR TITLE
updated Network.js

### PR DESCRIPTION
**Pull request by arjunvijayvargiya**
I updated Network.js in order to resolve privly-android issue privly/privly-android#41
This will cause the app to be redirected to apps Login Activity in case of authentication failure

### CODE REVIEW

File: shared/javascripts/network_service.js [hunk 1]
```diff
@@ -437,15 +437,19 @@ var privlyNetworkService = {
    * Show/hide the appropriate navigation items for when the user is logged out.
    */
   showLoggedOutNav: function() {
-    
-    // Don't show the nav at all if the content is injected.
-    if(typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
+  var isAndroid=privlyNetworkService.platformName();
```
Comment (smcgregor) [inline shared/javascripts/network_service.js]: inconsistent block indentation. Also, if there are tabs in this, you should remove them.

Comment (smcgregor) [inline shared/javascripts/network_service.js]: I don't believe "isAndroid" is defined.

Comment (smcgregor) [inline shared/javascripts/network_service.js]: yes, it is, but it is not being defined as a boolean.

**Comment by smcgregor** [comment]
Take another look at this then leave a comment for me to review again.

**Comment by arjunvijayvargiya** [comment]
Things done
1) removed extra variable for checking platform.
2)made indentations proper.

**Comment by smcgregor** [comment]
You changed indentation on the entire file, not just the part you committed.

**Comment by arjunvijayvargiya** [comment]
Ok I wil fix this and will let you know as soon as possible.

**Comment by arjunvijayvargiya** [comment]
I closed this because something wrong happened with my branch

**Pull request closed without merging**